packages/tests/shadydom/active-element.html

```diff
@@ -481,6 +481,7 @@
     r_1_1.focus();
     try {
       document.activeElement = "abc";
+    // eslint-disable-next-line no-empty
     } catch(e) {}
```

Comment:
```suggestion
    }
```

(also depends on polyserve)